packages/annotations/src/search.ts

```diff
@@ -144,7 +144,7 @@ export interface IVoteResourceObject {
  *     //   }]
  *    });
  * ```
- * Query for up and down votes on multiple comments from ArcGIS Hub.
+ * Query for up and down votes on a comment from ArcGIS Hub.
```

Comment:
we should probably use "annotation" instead of "comment" here and throughout the docs